### PR TITLE
Tux/crp/sessions controller specs

### PR DESCRIPTION
**Pull request by corinnepingul**
These are all passing tests.

### CODE REVIEW

File: spec/controllers/sessions_controller_spec.rb [hunk 1]
```diff
@@ -1,5 +1,99 @@
...
+    end
+  end
+
+  describe "DELETE #destroy" do
+    let(:seller_login_params) { { session: { username: "user1", password: "password1" } } }
+
+    before :each do
+      @seller = Seller.new(username: "user1", email: "email@email.com")
+      @seller.password, @seller.password_confirmation = "password1"
+      @seller.save
```
Comment (catchingash) [inline spec/controllers/sessions_controller_spec.rb]: You can condense this to 1 line as:

`@seller = Seller.new(username: "user1", email: "email@email.com", password: "password1", password_confirmation: "password1")`

But if you like this better, that's okay. Just let me know.

Comment (corinnepingul) [inline spec/controllers/sessions_controller_spec.rb]: No, that's great! I wasn't sure if that would work, but I can change it.

Comment (catchingash) [inline spec/controllers/sessions_controller_spec.rb]: Perfect. Whenever you push that for the couple of places you set `@seller`, then I'll merge it.

Comment (catchingash) [inline spec/controllers/sessions_controller_spec.rb]: Perfect. Once you change it, I'll merge this!

Comment (catchingash) [inline spec/controllers/sessions_controller_spec.rb]: Looks like you changed the one above, but not this one?

Comment (corinnepingul) [inline spec/controllers/sessions_controller_spec.rb]: Just finished the second change.

**Pull request merged**